calibrate/.env.sample

```diff
@@ -1,6 +1,6 @@
 PORT=5000
 NODE_PATH=src/
 REACT_APP_BASE_URL=http://staging-platform.airqo.net/api/v1/
-REACT_APP_DOCS_BASE_URL=https://staging-docs.airqo.net/#/
+REACT_APP_DOCS_BASE_URL=https://wiki.airqo.net/#/
```

Comment:
thanks @123MwanjeMike , do we also have `staging-wiki`?
If yes, shouldn't the `BASE_URL` be an ENV?